cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -91,17 +99,20 @@ def test_vm_console(request, appliance, setup_provider, context, configure_webso
                 # Enter Password:
                 vm_console.send_keys("{}".format(console_vm_password))
                 logger.info("Wait to get the '$' prompt")
-                vm_console.wait_for_text(text_to_find=catalog_item.provider.data.templates.
-                    console_template.prompt_text, timeout=200)
+                if provider.one_of(OpenStackProvider):
+                    time.sleep(15)
```

Comment:
**REQUIRED**: Anything we can wait for here instead of a sleep, we really dislike sleeping....

Reply:
There is no prompt in the Cirros templates, it just gives you "$" and the tesseract image-to-text not reliably read that, hence we can't try to match that. We can try to just get rid of sleep statement and rely on _validate_login() for validating login. I will try to see how that would work.